lib/reinteractive/style/version.rb

```diff
@@ -1,5 +1,5 @@
 module Reinteractive
   module Style
-    VERSION = "0.2.2".freeze
+    VERSION = "0.2.3".freeze
```

Comment:
I haven't updated the deployment documentation yet. However we should be using the git flow workflow. This means once you create a release then you bump the version here.

Reply:
Will merge like this and next time I'll do it properly